### PR TITLE
Now with two supercomputers: MET and UiO (Abel)

### PR DESCRIPTION
**Pull request by setmar**
These changes should be double checked.

### CODE REVIEW

File: gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb [hunk 1]
```diff
@@ -279,7 +306,8 @@
     "\n",
     "ax.bar(index, normalized_megacells_laptop, bar_width, label='Laptop')\n",
     "ax.bar(index+bar_width, normalized_megacells_desktop, bar_width, label='Desktop')\n",
-    "ax.bar(index+2*bar_width, normalized_megacells_supercomputer, bar_width, label='Supercomputer')\n",
```
Comment (havahol) [inline gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb]: Give the bar chart some air. I suggest:

```
bar_width = 0.2

ax.bar(index-1.5*bar_width, normalized_megacells_laptop, bar_width, label='Laptop')
ax.bar(index-0.5*bar_width, normalized_megacells_desktop, bar_width, label='Desktop')
ax.bar(index+0.5*bar_width, normalized_megacells_supercomputer_met, bar_width, label='Supercomputer MET')
ax.bar(index+1.5*bar_width, normalized_megacells_supercomputer_uio, bar_width, label='Supercomputer UiO')
```

File: gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb [hunk 2]
```diff
@@ -183,8 +200,11 @@
     "        if architecture == Architecture.DESKTOP:\n",
     "            block_size_options = [\"--block_width\", str(desktop_block[0]), \"--block_height\", str(desktop_block[1])]\n",
     "            \n",
```
Comment (havahol) [inline gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb]: Move
```block_size_options = None```
to above 
```if architecture == Architecture.DESKTOP or architecture == Architecture.SUPERCOMPUTER:```

File: gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb (hunk 1 above)
Comment (setmar) [reply gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb]: Used your suggestion.

File: gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb (hunk 2 above)
Comment (setmar) [reply gpu_ocean/prototypes/BenchmarkGitVersionsForProfiling.ipynb]: Fixed!

**Pull request merged**